tests/integration/plugins/test_changes.py

```diff
@@ -19,24 +20,29 @@ async def test_changes_plugin(
 
     anonymous_client = make_client(tuple())
     records = await anonymous_client.get_records(bucket="monitor", collection="changes")
+    assert records
+
+    test_collection_records = [
+        record for record in records if record["collection"] == source_collection
+    ]
+    assert len(test_collection_records) == 2
 
     editor_client = make_client(editor_auth)
     resource = await signed_resource(editor_client)
+    assert "bucket" in test_collection_records[0]
+    assert test_collection_records[0]["bucket"] == resource["destination"]["bucket"]
+    assert test_collection_records[1]["bucket"] == resource["preview"]["bucket"]
```

Comment:
These assertions are pretty flaky. Sometimes these assertions pass as-is, sometimes switching to

```python
    assert test_collection_records[0]["bucket"] == resource["preview"]["bucket"]
    assert test_collection_records[1]["bucket"] == resource["destination"]["bucket"]
```

is the thing that works.

@leplatrem any ideas here? Should this order actually matter? I also kind of don't know how 2 records are magically present in the collection before we upload any.

Reply:
In theory, the list of latest changes is sorted by `-last_modified`, and `destination` should be at `[0]`, and `preview` at `[1]`.
Because when we approve changes, we first copy the records to `preview`,  and then to `destination`.

If that is the only blocker, we could rewrite the assertion to remove notion of order and open an issue to fix the tests later.

Reply:
👍🏻 [24b589f](https://github.com/mozilla/remote-settings/pull/235/commits/24b589fc3ca9f462211014149ecbe7797d36ed01)

https://github.com/mozilla/remote-settings/issues/377